pkgs/development/compilers/purescript/purescript/default.nix

```diff
@@ -1,57 +1,53 @@
-{ stdenv, pkgs, fetchurl, zlib, gmp, lib }:
+{ stdenv
+, pkgs
+, lib
 
-# from justinwoo/easy-purescript-nix
-# https://github.com/justinwoo/easy-purescript-nix/blob/d383972c82620a712ead4033db14110497bc2c9c/purs.nix
+# set to true for a much smaller closure (2 builds instead of 121) at
+# the expense of risky version-constraint jailbreaking
+, useJailbreak ? false
```

Comment:
In my opinion, we should remove this flag and either provide the jailbroken or non-jailbroken build.  I think it will likely be too much work to maintain both builds, and most users will only ever use whatever we make the default.

We should probably just use the non-jailbroken build, since that is what upstream would likely prefer us to use.